living/malayalam.md

```diff
@@ -1,46 +1,46 @@
 # Web Language: Malayalam
 
-Additional names:
```

Comment:
Why was this section removed?

Reply:
I thought there were no additional names for the Malayalam language, so I removed that field. I will add it again if it is necessary.

Reply:
The name that was removed is `മലയാളം`

Reply:
This is useful if anyone wants to look up a language using a native name in the repository search, such as https://github.com/search?q=repo%3Acommoncrawl%2Fweb-languages%20%E0%B4%AE%E0%B4%B2%E0%B4%AF%E0%B4%BE%E0%B4%B3%E0%B4%82&type=code

Reply:
It’s actually not an additional name; it’s just the Malayalam word written in the Malayalam script. Should I add it?”

Reply:
Yes please!